### ISSUE TITLE
Report to check/add indexes

### ISSUE THREAD

**Issue by balamurali27** [frappe/press#1042]
Seems users may remove indexes at some point. Would be easy if we could check these without ssh.

**Comment by ankush** [comment]
This kinda belongs in Framework.

Any single column index that's added without adding in schmea will be lost after migrate during sync.

**Comment by balamurali27** [comment]
Yeah, makes sense for it to be in framework. Custom indexes feature ig
